src/apiProxy/api-proxy.js

```diff
@@ -99,7 +99,7 @@ function urlFormatParams(params, doEncode) {
  */
 export const buildRequestArgs = externalRequestOpts => ({ endpoint, params }) => {
 	const externalRequestOptsQuery = { ...externalRequestOpts };
-	externalRequestOptsQuery.url = `/${endpoint}`;
+	externalRequestOptsQuery.url = encodeURI(`/${endpoint}`);
```

Comment:
This seemed like the best place to make sure this is always encoded, and I can't think of a reason we wouldn't want to do this before sending along to the API.